tests/unit_tests/libs_tests/GameEntityManipulation_tests/CreateEntitiesFunctions_tests/CMakeLists.txt

```diff
@@ -8,10 +8,11 @@ set(TEST_SRC_ROOT
     ${SRC_ROOT}/CreatePlayer_test.cpp
     ${SRC_ROOT}/CreateEnemy_test.cpp
     ${SRC_ROOT}/CreateObstacle_test.cpp
+    ${SRC_ROOT}/CreateEnemyRandom_test.cpp
 )
 
 create_test(
     TEST_NAME create_entities_tests
     TEST_SRC ${TEST_SRC_ROOT}
-    TEST_DEPS ${PROJECT_NAME}_createEntitiesFunctions ${PROJECT_NAME}_ecs
+    TEST_DEPS ${PROJECT_NAME}_createEntitiesFunctions ${PROJECT_NAME}_ecs ${PROJECT_NAME}_gameSharedResources
```

Comment:
gameSharedRessources is not already link inside createEntitiesFunction ?

Reply:
Nop

Reply:
Do I have to put it with ?

Reply:
I don't know, put it for the moment i take a look later